server/src/main/java/org/elasticsearch/common/time/DateFormatters.java

```diff
@@ -1544,105 +1552,86 @@ static JavaDateFormatter merge(String pattern, List<DateFormatter> formatters) {
             dateTimeFormatters.toArray(new DateTimeFormatter[0]));
     }
 
-    private static final ZonedDateTime EPOCH_ZONED_DATE_TIME = Instant.EPOCH.atZone(ZoneOffset.UTC);
+    private static final LocalDate LOCALDATE_EPOCH = LocalDate.of(1970, 1, 1);
```

Comment:
there is an annotation below `    @SuppressForbidden(reason = "LocalDate.of is fine here")`
Why is it forbidden ? Should be also then applied here?